user/e16/update.py

```diff
@@ -0,0 +1,2 @@
+pattern = r"e16-(\d.\d.\d+)\.tar\.gz/"
```

Comment:
```suggestion
```

the default pattern works in this case

Reply:
OK great, I've removed the patter and tested the update check, it seems to work as it did before. Changes have been pushed to this branch.